app/dot/cpp/core/exceptions/FormException.java

```diff
@@ -8,7 +9,7 @@ public class FormException extends BaseException {
   private final transient List<ValidationError> formErrors;
 
   public FormException(List<ValidationError> formErrors) {
-    super("form.validation.errors");
+    super(ErrorCodes.FORM_VALIDATION_FAILED.getCode());
```

Comment:
Shouldn't this be getMessage() instead of getCode()?

Reply:
This is a bit confusing and I am open to suggestions on how to refactor this. ErrorCodes is an enum. The getCode method would return the ErrorCode class. That contains code, message and details.

Reply:
I didn't know what that super was actually, I checked the code now and it's fine.